hasura/metadata/actions.graphql

```diff
@@ -1,168 +1,116 @@
 type Mutation {
```

Comment:
I had the same issue the formatting here changed for some reason although i've update my hasura-cli version